jobs/university_groningen_22032016.html

```diff
@@ -0,0 +1,43 @@
+---
+title: Scientific Python Developer
+company: University of Groningen
+url: http://www.rug.nl
+location: Groningen, Netherlands
+contract: contrcat
```

Comment:
Spelling mistake on this item:

'contract' must contain one of: temp, part-time, other, perm, contract